### PR TITLE
Develop search-bar

### PR DESCRIPTION
**Pull request by shevabiz**
https://shevabiz.github.io/layout_search-bar-airbnb/
https://shevabiz.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,7 +10,29 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form action="/" method="get"
```
Comment (mykhalenych) [inline src/index.html:14]: general check html structure
Screenshot 2023-01-06 at 16 18 13

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  line-height: 22px;
+}
+
+.input_small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input[data-qa="keypress"] {
```
Comment (mykhalenych) [inline src/style.css:129]: better use classnames instead data attributes

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (shevabiz) [reply src/style.css]: Виправив зауваження

File: src/index.html (hunk 1 above)
Comment (shevabiz) [reply src/index.html]: Виправив зауваження

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,36 @@
...
+    class="search-form .search-form_big"
+    >
+      <label class="search-form_label search-form_label_big">
+        <input 
+      type="text" 
+      name="big-search" 
+      placeholder="Try &quot;Los Angeles&quot;" 
+      data-qa="keypress"
+      class="input input_big"
+        >
```
Comment (Anastasiia-Svintsova) [inline src/index.html:27]: Pay attention to your code style, please
```suggestion
        <input 
          type="text" 
          name="big-search" 
          placeholder="Try &quot;Los Angeles&quot;" 
          data-qa="keypress"
          class="input input_big"
        >
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    action="/" 
+    method="get" 
+    data-qa="big"
+    class="search-form .search-form_big"
+    >
```
Comment (Anastasiia-Svintsova) [inline src/index.html:19]:
```suggestion
    <form 
      action="/" 
      method="get" 
      data-qa="big"
      class="search-form .search-form_big"
    >
```

File: src/index.html [hunk 5]
```diff
@@ -11,6 +11,36 @@
...
+    method="get" 
+    data-qa="small"
+    class="search-form search-form_small">
+      <label class="search-form_label search-form_label_small">
+        <input 
+        type="text" 
+        name="small-search" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input input_small"
+        >
```
Comment (Anastasiia-Svintsova) [inline src/index.html:42]: Same here

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  font-style: normal;
+  font-weight: 600;
+
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+ 
+}
```
Comment (Anastasiia-Svintsova) [inline src/style.css:25]: Check it everywhere
```suggestion
  font-size: 16px;
}
```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+.search-form_label_big:after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:66]: Check your properties order, please https://9elements.com/css-rule-order/

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+.search-form_label_small:after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:72]: Combine common styles for labels

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 7 above)
Comment (mykhalenych) [reply src/style.css:60]: didn't fix it

File: src/style.css (hunk 8 above)
Comment (mykhalenych) [reply src/style.css:66]: the same

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 8 above)
Comment (shevabiz) [reply src/style.css]: Fixed the errors

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
```
Comment (OlEzhA0) [inline src/style.css:2]: why do we need to reset padding for `*` selector? reset only where needed

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
```
Comment (OlEzhA0) [inline src/style.css:27]: it's redundant

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form_label {
+  position: relative;
+  display: block;
+  width: 100%;
```
Comment (OlEzhA0) [inline src/style.css:44]: it's redundant

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+
+.search-form_label_small:after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
```
Comment (OlEzhA0) [inline src/style.css:71]:
```suggestion
.search-form_label:after {
  position: absolute;
  content: "";
  display: block;
  background-image: url("images/Search.svg");
  background-repeat: no-repeat;
  background-size: contain;
}

.search-form_label_big:after {
  top: 25px;
  left: 26px;
  width: 19px;
  height: 19px;
}

.search-form_label_small:after {
  top: 15px;
  left: 13px;
  width: 11px;
  height: 11px;
}
```

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 600;
+  border-radius: 3px;
+}
+
+input::placeholder {
```
Comment (OlEzhA0) [inline src/style.css:108]: don't style tags

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 11 above)
Comment (shevabiz) [reply src/style.css:44]: after fixing the edits does not work correctly

File: src/style.css (hunk 10 above)
Comment (shevabiz) [reply src/style.css:27]: after fixing the edits does not work correctly

File: src/style.css (hunk 11 above)
Comment (shevabiz) [reply src/style.css:44]: after fixing the edits does not work correctly

File: src/style.css (hunk 10 above)
Comment (shevabiz) [reply src/style.css:27]: after fixing the edits does not work correctly

File: src/style.css (hunk 9 above)
Comment (shevabiz) [reply src/style.css:2]: after fixing the edits does not work correctly

File: src/style.css (hunk 13 above)
Comment (shevabiz) [reply src/style.css:105]: after fixing the edits does not work correctly

File: src/style.css (hunk 10 above)
Comment (witflash) [reply src/style.css:25]: Це правило відносилось до flex box і зараз воно просто ніяку роль не відіграє, просто займає зайве місце в коді :)

**Review by witflash — APPROVED** [review]
👌

**Pull request closed without merging**